src/docfx/build/toc/BuildTableOfContents.cs

```diff
@@ -12,5 +14,22 @@ public static Task Build(Context context, Document file)
         {
             throw new NotImplementedException();
         }
+
+        public static Task BuildTocMap(Context context, Document fileToBuild, TableOfContentsMapBuilder tocMapBuilder)
+        {
+            Debug.Assert(tocMapBuilder != null);
+            Debug.Assert(fileToBuild != null);
+
+            var (tocModel, referencedDocuments, referencedTocs) = Load(fileToBuild);
+
+            tocMapBuilder.Add(fileToBuild, referencedDocuments, referencedTocs);
+
+            return Task.FromResult(0);
```

Comment:
Task.CompletedTask?

Reply:
This method is not even an async method, so just `void BuildTocMap`

Reply:
i don't know the parse toc is void or task, so just leave it as task, will change it later

---
In reply to: [184586840](https://github.com/dotnet/docfx/pull/2696#discussion_r184586840) [](ancestors = 184586840)